testsuite/tests/ui/policies/test_policies.py

```diff
@@ -118,3 +118,36 @@ def test_move_policy(login, navigator, policy_service, api_client, policy_applic
 
     response = api_client(app=policy_application).get('/anything')
     assert response.status_code == 333
+
+
+# pylint: disable=unused-argument
+def test_edit_policy_widgets(login, navigator, policy_service):
+    """
+    Test:
+        - Create service via API
+        - Navigate to Policies page and add URL Rewriting policy with rewriting command via UI
+        - Assert that policy was added and is applied
+        - Navigate to the configuration page and promote new configuration to the staging phase
+        - Fetch product policies
+        - Assert that new URL Rewriting policy configuration was successfully applied
+    """
+    policies_page = navigator.navigate(ProductPoliciesView, product=policy_service)
+    policies_page.add_policy(Policies.URL_REWRITING.value)
+    assert policies_page.policy_section.has_item(Policies.URL_REWRITING.value)
+
+    policies_page.policy_section.edit_policy(Policies.URL_REWRITING.value)
+    policies_page.url_rewrite_policy_view.add_rewriting_command(regex="hello", replace="get", op="gsub")
```

Comment:
calling this method require user to choose between just two strings: "gsub" and "sub". 
If the user writes here some other string, nothing will be selected in the relevant select widget. In addition, he would need to manually check the Dome object to find what values are required. Please try to find a different solution for this parameter.

Reply:
Add 'Literal' type hint to the validation of the parameters. Will that be enough?